packages/core/src/App/app.jsx

```diff
@@ -66,8 +66,8 @@ const AppWithoutTranslation = ({ root_store }) => {
     const handleResize = React.useCallback(() => {
         if (isTouchDevice() && (isMobile() || isTablet())) {
             const is_android_device = mobileOSDetect() === 'Android';
-            const view_width = is_android_device ? screen.availWidth : window.innerWidth;
-            const view_height = is_android_device ? screen.availHeight : window.innerHeight;
+            const view_width = is_android_device ? screen.availWidth : screen.width;
+            const view_height = is_android_device ? screen.availHeight : screen.height;
```

Comment:
Need to check screen.width & height instead of window innerWidth  & innerHeight, as the keyboard on safari is messing with the viewport sizes, making the screen display landscape-blocker component. screen will be measuring the entire screen's resolution, hopefully this will combat the problem on iOS safari keyboard issue